src/main/java/seedu/address/storage/JsonSerializableFriendlyLink.java

```diff
@@ -20,15 +21,20 @@
 class JsonSerializableFriendlyLink {
 
     public static final String MESSAGE_DUPLICATE_PERSON = "Persons list contains duplicate person(s).";
+    public static final String MESSAGE_DUPLICATE_PAIR = "Persons list contains duplicate pair(s).";
 
     private final List<JsonAdaptedPerson> persons = new ArrayList<>();
 
+    private final List<JsonAdaptedPair> pairs = new ArrayList<>();
+
     /**
      * Constructs a {@code JsonSerializableFriendlyLink} with the given persons.
      */
     @JsonCreator
-    public JsonSerializableFriendlyLink(@JsonProperty("persons") List<JsonAdaptedPerson> persons) {
+    public JsonSerializableFriendlyLink(@JsonProperty("persons") List<JsonAdaptedPerson> persons,
+                                       @JsonProperty("pairs") List<JsonAdaptedPair> pairs) {
```

Comment:
Same as above

Reply:
Fixed in fd3291c77aaaa7842817433d8a56d30c145f5ac5